apps/src/templates/accessible-dialogue.module.scss

```diff
@@ -0,0 +1,24 @@
+.modalBackdrop {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background-color: #000;
+  opacity: 0.6;
+  z-index: 1000;
+};
+
+.modal {
+  position: fixed;
+  top: 10%;
+  left: 50%;
+  transform: translateX(-50%);
+  z-index: 1050;
+  width: 70%;
+  max-width: 600px;
+  background-color: #fff;
+  border: 1px solid rgba(0 0 0 / 0.3);
+  border-radius: 4px;
+  padding: 15px;
```

Comment:
@megcrenshaw you can use `1rem` here 🎉 — this will compute to 16px

Reply:
Hurray! Thanks